Dockerfile

```diff
@@ -65,8 +65,7 @@ RUN /bin/bash -c 'TMP_PKG_DIR=$(mktemp -d); \
         | xargs -r apk info --installed \
         | sort -u \
     )" \
- && apk add --virtual .rundeps $runDeps \
- && apk del .build-deps
```

Comment:
Couple of other packages should be installed here to facilitate extension install as noted from your comment:
- gcc 
- make 
- openssl-dev 
- libffi-dev 
- musl-dev

Reply:
Derp, I'm blind. Resolving this comment.